### PR TITLE
[Refactor] Volume Group Action

### PR DESCRIPTION
**Pull request by michaelkad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by ismirlia** [comment]
```
VolumeGroup Action Test
--- PASS: TestAccIBMPIVolumeGroupActionbasic (398.17s)
PASS
```

**Review by Alexander-Kita — APPROVED** [review]
Looks good, but fix conflicts before moving over.

File: ibm/service/power/resource_ibm_pi_volume_group_action.go [hunk 1]
```diff
@@ -30,90 +30,99 @@ func ResourceIBMPIVolumeGroupAction() *schema.Resource {
...
+				ValidateFunc: validation.NoZeroValues,
 			},
-			PIVolumeGroupAction: {
-				Type:        schema.TypeList,
-				Required:    true,
-				ForceNew:    true,
-				MaxItems:    1,
-				MinItems:    1,
+			Arg_VolumeGroupAction: {
 				Description: "Performs an action (start stop reset ) on a volume group(one at a time).",
```
Comment (Alexander-Kita) [inline ibm/service/power/resource_ibm_pi_volume_group_action.go:42]:
```suggestion
				Description: "Performs an action (start stop reset) on a volume group (one at a time).",
```

File: ibm/service/power/resource_ibm_pi_volume_group_action.go [hunk 2]
```diff
@@ -30,90 +30,99 @@ func ResourceIBMPIVolumeGroupAction() *schema.Resource {
...
-							Type:     schema.TypeList,
-							Optional: true,
-							MaxItems: 1,
 							ForceNew: true,
+							MaxItems: 1,
+							Optional: true,
+							Type:     schema.TypeList,
+						},
+						Attr_Reset: {
+							Description: "Performs reset action on the volume group to update its  value.",
```
Comment (Alexander-Kita) [inline ibm/service/power/resource_ibm_pi_volume_group_action.go:79]:
```suggestion
							Description: "Performs reset action on the volume group to update its value.",
```

File: ibm/service/power/resource_ibm_pi_volume_group_action.go (hunk 1 above)
Comment (ismirlia) [reply ibm/service/power/resource_ibm_pi_volume_group_action.go]: fixed

File: ibm/service/power/resource_ibm_pi_volume_group_action.go (hunk 2 above)
Comment (ismirlia) [reply ibm/service/power/resource_ibm_pi_volume_group_action.go]: fixed

**Review by Alexander-Kita — APPROVED** [review]

**Comment by ismirlia** [comment]
Opened: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5780 closing this one.

**Pull request closed without merging**